docs/B-Computations/style-guidelines.md

```diff
@@ -1,5 +1,8 @@
 ---
 sidebar_position: 5
+title: "Style-Guidelines"
+id: "style-guidlines"
+slug: "/B-Computations/style-guidelines"
```

Comment:
```suggestion
slug: /computations/style-guidelines
```
To keep it consistent with the standardized frontmatter. As referenced in issue #64 .

Reply:
Thank you for the reference to #64. I have updated my code.